src/ray/gcs/gcs_server/store_client_kv.cc

```diff
@@ -62,6 +62,7 @@ void StoreClientInternalKV::Get(
       table_name_,
       MakeKey(ns, key),
       [callback = std::move(callback)](auto status, auto result) {
+        RAY_CHECK(status.ok()) << "Failed to get from Redis with status: " << status;
```

Comment:
Do you know why the redis client doesn't crash?

We probably shouldn't add it here. Try to put it into lower part.

Reply:
Good point